api/migrations/20200626194403-removeOrphanedDocs.js

```diff
@@ -0,0 +1,94 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
```

Comment:
No vars - here and below

Reply:
3fb0c0014b44f40a4f53dd1c437c15124582029f